spec/support/order_process_tag_links_helper.rb

```diff
@@ -0,0 +1,28 @@
+RSpec.shared_context "tag links of an order process with parameters set" do
+  let(:order_process_tag) do
+    {:tag => "/#{Api::V1x2::Catalog::TaggingService::TAG_NAMESPACE}/#{Api::V1x2::Catalog::TaggingService::TAG_NAME}=#{order_process.id}"}
+  end
+
+  let(:order_process_tags) { [order_process_tag] }
+  let(:http_status) { [200, 'Ok'] }
+  let(:headers)     do
+    {'Content-Type' => 'application/json'}.merge(default_headers)
+  end
+
+  let(:test_env) do
+    {
+      :TOPOLOGICAL_INVENTORY_URL => 'http://localhost',
+      :CATALOG_URL               => 'http://localhost',
+      :SOURCES_URL               => 'http://localhost'
```

Comment:
I believe everywhere else where we've used these constants we went with something like `http://topology.example.com/`, `http://catalog.example.com/`, `http://sources.example.com/`, etc, so I think these should probably change just to remain consistent. You will probably have to change your `stub_requests` to match the proper URL as well.